datastore/store.go

```diff
@@ -38,6 +38,10 @@ type MultiStore interface {
 	// under the /data namespace
 	Datastore() DSReaderWriter
 
+	// Encstore is a wrapped root DSReaderWriter
+	// under the /enc namespace
+	Encstore() DSReaderWriter
```

Comment:
todo: I know the documentation for the other members of this interface isn't great, but their names are a little more self explanatory.  Can you please document what this store is?  And perhaps rename it to something a little clearer - `Enc` is not really self documenting outside the context of this PR, and this is a public type.

Reply:
I'm not 100% happy myself with the name, but didn't find any better alternative. `EncryptionStore` seems to be too long. Fred [proposed](https://github.com/sourcenetwork/defradb/pull/2731#discussion_r1643153189) `CryptoStore`. Feel free to propose something better.

Reply:
I'm happy with the length of `EncryptionStore`, although I'm not really happy with that name anyway (is still vague and is probably more likely to be read as a store of encrypted data than a store of keys).

For me the most important is the documentation request - please see the name change as a suggestion.

Reply:
added documentation